orchestra/communication/staffing.py

```diff
@@ -122,48 +127,130 @@ def check_responses_complete(request):
              .format(request.task)))
 
 
-def send_staffing_requests(
+def address_staffing_requests(
         worker_batch_size=settings.ORCHESTRA_STAFFBOT_WORKER_BATCH_SIZE,
         frequency=settings.ORCHESTRA_STAFFBOT_BATCH_FREQUENCY):
     staffbot = StaffBot()
     cutoff_datetime = timezone.now() - frequency
     requests = (
         StaffBotRequest.objects
-        .filter(status=StaffBotRequest.Status.SENDING_INQUIRIES.value)
+        .filter(status__in=[
+            StaffBotRequest.Status.SENDING_INQUIRIES.value,
+            StaffBotRequest.Status.DONE_SENDING_INQUIRIES.value])
         .filter(Q(last_inquiry_sent__isnull=True) |
-                Q(last_inquiry_sent__lte=cutoff_datetime)))
+                Q(last_inquiry_sent__lte=cutoff_datetime))
+        .order_by('-task__project__priority', 'created_at'))
 
     for request in requests:
-        send_request_inquiries(staffbot, request, worker_batch_size)
+        staff_or_send_request_inquiries(staffbot, request, worker_batch_size)
+
+
+def _is_worker_assignable(worker, task, required_role):
+    try:
+        check_worker_allowed_new_assignment(worker)
+        if (is_worker_certified_for_task(worker, task,
```

Comment:
if we care, can return this rather than if X: return True

Reply:
"we don't care"